package.json

```diff
@@ -66,9 +66,14 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "testem ci",
-    "lint-dts": "tslint sweetalert2.d.ts --format verbose",
-    "check-dts": "tsc sweetalert2.d.ts && npm run lint-dts",
+    "start": "gulp watch",
```

Comment:
indicates the command to get `start`ed (as standard package script) and doesn't require devs to have gulp globally installed

Reply:
Note that you can invoke this with just `npm start`

Reply:
:+1: